lib/autokey/scripting/system.py

```diff
@@ -43,11 +43,10 @@ def exec_command(command, getOutput=True):
                     stdout=subprocess.PIPE,
                     universal_newlines=True) as p:
                 output = p.communicate()[0]
-                if output.endswith("\n"):
+                if len(output) > 0:
```

Comment:
Good advice @mvindiola1. @sebastiansam55 if you just remove this conditional, then @josephj11 should merge this change.

As an aside, I couldn't replicate this issue with automated tests, so if you can figure out how to, add the test to `autokey/tests/scripting_api/test_system.py`

Reply:
@sebastiansam55 can you quickly please replace lines 46-49 with

```python
# Most shell output has a new line at the end, which we don't want.
output = output.rstrip("\n")
```

Then repush and we can merge. This has cropped up as a bug elsewhere.